Microsoft.Toolkit.Sample.Forms.App/Program.cs

```diff
@@ -18,9 +16,15 @@ static class Program
         [STAThread]
         static void Main()
         {
-            Application.EnableVisualStyles();
-            Application.SetCompatibleTextRenderingDefault(true);
-            Application.Run(new Form1());
+            using (new UI.XamlHost.XamlApplication()
+            {
+                RequestedTheme = WUX.ApplicationTheme.Dark,
+            })
+            {
+                Application.EnableVisualStyles();
+                Application.SetCompatibleTextRenderingDefault(true);
+                Application.Run(new Form1());
+            }
```

Comment:
I've noticed sometimes that WER is running after I close the sample app.
`Faulting application name: Microsoft.Toolkit.Sample.Forms.App.exe, version: 6.0.0.34, time stamp: 0x5c49f55f
Faulting module name: Windows.UI.Xaml.dll, version: 10.0.18343.1, time stamp: 0x9cb6f86d
Exception code: 0xc000027b
Fault offset: 0x0000000000713030
Faulting process id: 0x8868
Faulting application start time: 0x01d4cd34384863fd
Faulting application path: C:\Users\alzollin\source\repos\Microsoft.Toolkit.Win32\Microsoft.Toolkit.Sample.Forms.App\bin\AnyCPU\Debug\netcoreapp3.0\Microsoft.Toolkit.Sample.Forms.App.exe
Faulting module path: C:\Windows\System32\Windows.UI.Xaml.dll
Report Id: 41d7acb4-aee1-421e-a66b-44f56071e12a
Faulting package full name: 
Faulting package-relative application ID: `
Not only for .Net Core, but also for net462.

Reply:
$crashFolder = 'c:\CrashDumps'
$regKey = 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps'
if ( !(test-path $regKey) )
{
    mkdir $regKey
}
if (!(test-path $crashFolder))
{
  mkdir $crashFolder
}
set-itemproperty $regKey DumpFolder $crashFolder -type ExpandString
set-itemproperty $regKey DumpCount 10 -type dword
set-itemproperty $regKey DumpType 2 -type dword

Please run that powershell, run the scenario again and share the dump file.

Reply:
Confirmed that the in the newer 18346 the bug does not repro anymore and that build contain specific bug fixes around tear down of the Xaml Island in Win32.


---
In reply to: [259948793](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r259948793) [](ancestors = 259948793)